src/Mixins/StoreCollection.php

```diff
@@ -37,8 +43,18 @@ public function collection()
                 {
                     return $this->collection;
                 }
+
+                /**
+                 * @return array
+                 */
+                public function headings(): array
+                {
+                    return $this->withHeadings ? $this->collection->collapse()->keys()->all() : [];
+                }
             };
 
+            $export->withHeadings = $withHeadings;
```

Comment:
I would prefer this parameter being passed through the constructor (see `new class($this)`)